frontend/src/views/Viewlevelstable.vue

```diff
@@ -0,0 +1,73 @@
+<template>
+  <div class="PageWrapper">
+    <div class="TableContainer">
+      <h2>Levels:</h2>
+      <br />
+      <ElTable :data="levels" style="width: 100%">
+        <ElTableColumn prop="name" label="Levels" />
+      </ElTable>
+      <div class="NewLevel">
+        <ElButton type="primary" @click="redirectToLevelView">Create Level</ElButton>
+      </div>
+    </div>
+  </div>
+</template>
+
+<script setup lang="ts">
+import { ref, onMounted } from 'vue'
+import { useRouter, useRoute } from 'vue-router'
+import { ElTable, ElTableColumn, ElButton } from 'element-plus'
+import { useLevelsStore } from '@/stores/levels'
+import { fetchLevelsByRoleId } from '@/stores/levels/actions'
+import { getLevels } from '@/stores/levels/getters'
```

Comment:
Line 21 and 22 you are using store functions directly, and this is not supposed to be done. Use levelsStore.functionYouWantToUse